Source/WebKit/UIProcess/FrameLoadState.cpp

```diff
@@ -46,8 +46,6 @@ void FrameLoadState::removeObserver(Observer& observer)
 
 void FrameLoadState::didStartProvisionalLoad(const URL& url)
 {
-    ASSERT(m_provisionalURL.isEmpty());
```

Comment:
I don't think this assertion is invalid. We hit it once in a while and every time it is a valid bug.

Seems to indicate duplicate didStartProvisionalLoad calls or a provisional load starting without the previous provisional load being cancelled first.